source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql

```diff
@@ -411,7 +379,6 @@ BEGIN TRY
 	WHERE rr.SourceSystem = 'NTBS'
 
 	DROP TABLE #TempDiseaseSites
-	DROP TABLE #TempManualTestResult
 	DROP TABLE #TempSocialContextVenues
 
 	--'Sample taken' should be set if there are any manually-entered test results which are NOT of type chest x-ray
```

Comment:
I've spotted this comment too, should we be also removing Chest CT results when looking for SampleTaken?

Reply:
I assume that we should, but I'm not certain. @adilmirzaphe What do you think?

Reply:
@JoshCadney  vaguely recall this being discussed with Esther. Might be worth clarifying in catch up meeting tomorrow.

Reply:
I'm going to put it in this ticket in JIRA, as it's a small change and still makes sense in this ticket.